go/oasis-node/cmd/debug/fixgenesis/fixgenesis.go

```diff
@@ -301,27 +324,57 @@ func updateGenesisDoc(oldDoc *oldDocument) (*genesis.Document, error) {
 		KeyManager: oldDoc.KeyManager,
 		Scheduler:  oldDoc.Scheduler,
 		Beacon:     oldDoc.Beacon,
-		Consensus:  oldDoc.Consensus,
 		HaltEpoch:  oldDoc.HaltEpoch,
 		ExtraData:  oldDoc.ExtraData,
 	}
 
+	// Consensus.
+	newDoc.Consensus.Backend = oldDoc.Consensus.Backend
+	newDoc.Consensus.Parameters = oldDoc.Consensus.Parameters.Parameters
+	// TODO: different constant?
+	newDoc.Consensus.Parameters.MaxEvidenceSize = 100 * uint64(oldDoc.Consensus.Parameters.MaxEvidenceNum)
```

Comment:
Probably should be 1024 to be safe.

Reply:
Just 1024 or `1024 * existing_MaxEvidenceNum`?